main.rb

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env ruby
+require './app'
+require 'date'
+
+class Main
+  def initialize()
+    @my_app = App.new
+  end
+
+  def run
+    puts 'Welcome to our app'
+    loop do
+      display_options
+      option = gets.chomp.to_i
+      handle_option(option)
+      break if option == 7
+    end
+    puts 'Exiting......'
+  end
+
+  def display_options
+    puts 'Please choose an option: '
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+  end
+
+  def handle_option(option)
+    case option
+    when 1
+      @my_app.list_books
+    when 2
+      @my_app.list_people
+    when 3
+      handle_create_person
+    when 4
+      handle_create_book
+    when 5
+      handle_create_rental
+    when 6
+      handle_list_rentals
+    else
+      puts 'Invalid option'
+    end
+  end
+
+  def handle_create_person
+    puts 'Choose a person: Teacher(1), Student(2). Enter the number.'
+    op_person = gets.chomp.to_i
+    case op_person
+    when 1
+      create_teacher
+    when 2
+      create_student
+    else
+      puts 'Invalid option'
+    end
+  end
+
+  def create_teacher
+    puts 'Enter Teachers Age: '
+    age = gets.chomp.to_i
+    puts 'Enter Teachers name: '
+    name = gets.chomp.to_s
+    puts 'Enter Specialization: '
+    specialization = gets.chomp.to_s
+    @my_app.create_person('Teacher', age, name, specialization)
+  end
+
+  def create_student
+    puts 'Enter students age: '
+    age = gets.chomp.to_i
+    puts 'Enter students name: '
+    name = gets.chomp.to_s
+    @my_app.create_person('Student', age, name)
+  end
```

Comment:
- [ ]  Hi @SimonGideon, great job your app is working. But you should make sure our school library console app behaves in the same way as in the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) video that was provided to us. 
- 1. When you create a person, the student (1) should be the first option and the teacher should be the second option (2) and not the inverse,
- 2. Make sure that in order to create the student  the app should request:
   - Age
   - Name
   - Has parent permission
- Kindly address this 💯 👍🏻  here is the screenshot of your console 👇🏻 

![image](https://user-images.githubusercontent.com/79058364/219854413-dca47644-c24f-4b76-972c-e1885afd4508.png)

Reply:
@BenMukebo  Noted the requested changes are implemented. Thanks for this :muscle: :slightly_smiling_face: